pkg/apb/mock_Executor.go

```diff
@@ -0,0 +1,134 @@
+// Code generated by mockery v1.0.0
```

Comment:
I am wondering if we should have a mock package for this? 

Just makes it more clear in your import that you're importing a mock?

Reply:
I started thinking about this as well, but it implies each pkg would have a corresponding child mock package? Does that make sense?

Reply:
what if you were to create a top level mock package that contained shared mocks instead of a mock package in each individual package?

Reply:
I don't think we want to be mixing mocks for a bunch of different packages into one package.